src/models/azureProvider.ts

```diff
@@ -18,3 +18,9 @@ export interface FunctionMetadata {
   handler: string;
   events: FunctionEvent[];
 }
+
+export interface DeploymentExtendedError {
```

Comment:
Why `DeploymentExtendedErorr`?  are we using `DeploymentError` already elsewhere?

Reply:
This is just an augmentation of the `DeploymentExtended` type (not defined by us). It has an `error` property that is not specified in the type.